isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -1967,8 +1970,8 @@ namespace Isis {
 
       if (m_bundleSettings->solveTargetBody()) {
         // TODO: needs to be updated for ISIS vs. CSM CSM has no updateBodyRotation]
-        // TODO: this is no good. 
-        QSharedPointer<BundleObservation> isisObservation = qSharedPointerDynamicCast<BundleObservation>(observation);  
+        // TODO: this is no good.
```

Comment:
In a future PR, these TODOs should be removed. The second one was mine and is not specific enough to be useful.

Reply:
(to remove the first TODO, I guess this should probably also throw an error (or perform some other expected behavior) if this is attempted with a CSM sensor.)